src/components/IndividualArticle.jsx

```diff
@@ -10,7 +10,24 @@ const IndividualArticle = () => {
     const [article, setArticle] = useState({})
     const [isLoading, setIsLoading] = useState(true)
     const [comments, setComments ] = useState([])
-    
+    const [newComment, setNewComment ] = useState("");
+
+
+
+    const onSubmit = event => {
```

Comment:
I don't think you need the event as a parameter here as you don't use it as far as I can tell? Could just write `const onSubmit = () => {...`